tests/src/test/scala/whisk/core/controller/test/ActionsApiTests.scala

```diff
@@ -138,7 +138,7 @@ class ActionsApiTests extends ControllerTestCommon with WhiskActionsApi {
     Get(s"$collectionPath/${action.name}") ~> Route.seal(routes(creds)) ~> check {
       status should be(OK)
       val response = responseAs[WhiskAction]
-      response should be(action)
+      response.toJson should be(action.toJson)
```

Comment:
Direct comparison of the WhiskAction here fails. However, comparing the actions as JSON seems to work.